src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/hotshard/HotShardClusterRca.java

```diff
@@ -54,8 +53,7 @@ public class HotShardClusterRca extends Rca<ResourceFlowUnit<HotClusterSummary>>
 
     // Guava Table with Row: 'Index_Name', Column: 'NodeShardKey', Cell Value: 'Value'
     private Table<String, NodeShardKey, Double> cpuUtilizationInfoTable;
```

Comment:
Within this new implementation, each node will be sending only topK*2 consumers to the cluster manager node and so the size of table is fixed. We can do further optimization with this, let us add a TODO for it.